transport-native-unix-common/src/main/java/io/netty/channel/unix/Socket.java

```diff
@@ -46,7 +46,8 @@
  */
 public class Socket extends FileDescriptor {
 
-    public static final int UDS_SUN_PATH_SIZE = udsSunPathSize();
+    @Deprecated
+    public static final int UDS_SUN_PATH_SIZE = 100;
```

Comment:
This was only used in a test and it is problematic as we may not have registered the methods yet when we try to call this